index.js

```diff
@@ -1,30 +1,51 @@
+const multiRet = ['push', 'pop', 'shift', 'unshift']
```

Comment:
I appreciate the motivation behind the change to `const`, however would prefer to keep using `let` simply to keep the size down (part of writing this lib was a challenge as to how small I could make it!).